molecule/default/molecule.yml

```diff
@@ -9,10 +9,15 @@ lint: |
     flake8
 platforms:
   - name: nfs-share
-    image: centos/systemd
+    image: eniocarboni/docker-rockylinux-systemd:9
```

Comment:
Just as a heads up which also occurred to me while reviewing https://github.com/ome/docker-example-omero/issues/13 is that we probably should have a general handle on accounts for whom we use docker images, either as a list or guidelines on yes&no.

Reply:
This is a simple one that we could easily handled ourselves if we think it is preferable